azure-pipelines.yml

```diff
@@ -69,6 +70,9 @@ jobs:
   - script: bundle exec bin/ci/cucumber.rb
     displayName: "exec cucumber"
 
+  - script: bundle exec bin/ci/appcenter.rb
```

Comment:
hi @jmoody, In my opinion, cucumber tests should be run before App Center Test Cloud because they are run on simulators and our hosted machine but Test Cloud uses real devices and external dependency (App Center Test Cloud service)

Reply:
The results of the tests against real devices are more important.

If we have bad results against physical devices, I would delay a DeviceAgent release (for example).